examples/plot_03_competing_risks.py

```diff
@@ -0,0 +1,250 @@
+"""
+==============================
+Exploring the accuracy in time
+==============================
+
+In this notebook, we showcase how the accuracy in time metric is defined, behaves, and
+how to interpret it.
+"""
+
+# %%
+# Definition of the Accuracy in Time
+# ==================================
+# Here is a little bit of context about this metric introduced in [Alberge2024]_:
+#
+# - It is a generalization of the accuracy metric in the survival and the competing
+#   risks setting, which is the proportion of correct predictions (e.g. highest
+#   predicted probability) at a fixed time.
+# - It is computed for different time horizons, given by the user, with direct time
+#   stamps or quantiles of the observed durations.
+# - For a given patient at a fixed time, we compare the actual observed event to the
+#   most predicted one. The censored patients are removed from the computation after
+#   their censoring duration. In other word, let's imagine a patient who has
+#   experienced death by cancer at time :math:`t`. Before this time, we want the model
+#   to predict with the highest probability that this patient will survive. After
+#   :math:`t`, we want the model to predict the death by cancer event with the highest
+#   probability.
+# - The mathematical formula is:
+#
+# .. math::
+#     \mathrm{acc}(\zeta) = \frac{1}{n_{nc}} \sum_{i=1}^n ~ I\{\hat{y}_i=y_{i,\zeta}\}
+#        \overline{I\{\delta_i = 0 \cap t_i \leq \zeta \}}
+#
```

Comment:
wouldn't it be clearer to write the second event as the opposite event instead of putting a bar over it to telle the opposite?

Reply:
and maybe say that I is the indicator event